lib/db/set.go

```diff
@@ -37,8 +37,12 @@ type FileSet struct {
 type FileIntf interface {
 	FileSize() int64
 	FileName() string
+	FileLocalFlags() uint32
```

Comment:
It's already a file interface, it's clear that local flags belong to a file. If it conflicts with the field then do a java and call it GetLocalFlags or LocalFlagsSet

Reply:
That's not what we did with the two attributes above it, `FileSize` and `FileName`.